tests/component/sections/shared/add-data-actions/AddDataActionsButton.spec.tsx

```diff
@@ -21,11 +21,61 @@ describe('AddDataActionsButton', () => {
 
   it('renders the new dataset button with the correct generated link', () => {
```

Comment:
```suggestion
  it('renders the new dataset button with the correct generated link for specified collectionId', () => {
```
(description was duplicate)